root/components/Table.js

```diff
@@ -14,9 +14,7 @@ import loopParity from '../utility/loopParity';
 
 const renderTableHeaderCell = (column) => (
   <th
-    {...column.getHeaderProps(
-      {...column.headerProps, className: column.className},
-    )}
```

Comment:
Added `cellProps` looks good, but I don’t get why `className` has been removed. It seems that no change in parameters should be required but to `lengthColumn`.

Reply:
The `className` appears to be passed through `headerProps` in all cases now, so that a special case isn't needed AFAICT.

Reply:
It just seems much more consistent to me compared with passing className in two different ways and having to check className: column.headerProps.className || column.className.

Reply:
From the below discussion, it seems that classes should be set only to headers in most cases, thus there won’t be that much duplicates between `headerProps` and `cellProps` anyway.